data/jobs.txt

```diff
@@ -1262,6 +1262,302 @@ mission "Escort (Extra Large, Southern, Dangerous Origin or Destination)"
 		dialog "The captain of the <npc> thanks you for escorting them safely, and pays you <payment>."
 	on visit
 		dialog "You have reached <planet>, but you left the <npc> behind! Better depart and wait for them to arrive in this star system."
+		
+mission "Rescue (Interceptor, Northern)"
+	name "Rescue Merchant Interceptor"
+	description "The captain of the <npc> bit off more than they could chew. Repair them and escort them safely back to <destination>."
+	repeat
+	job
+	source
+		government Republic "Free Worlds" Syndicate Neutral
+		attributes frontier north
+	to offer
+		random < 40
+		"combat rating" >= 20
+	
+	npc accompany save
+		system
+			distance 1 3
+		personality waiting derelict timid
```

Comment:
`waiting` is inconsequential as a personality since these are disabled ships, and they don't spawn in your system. The purpose of `waiting` is that the ship is "waiting in orbit" instead of taking off alongside you.